krun.py

```diff
@@ -140,13 +125,16 @@ class ScheduleEmpty(Exception):
 class ExecutionScheduler(object):
     """Represents our entire benchmarking session"""
 
-    def __init__(self, config_file, log_filename, out_file, mailer, platform):
+    def __init__(self, config_file, log_filename, out_file, mailer, platform,
```

Comment:
The number of args to this constructor has bugged me in the past. Perhaps we should wrap these into a class? Thoughts?